components/console-backend-service/internal/domain/k8s/selfsubjectrules_service.go

```diff
@@ -0,0 +1,135 @@
+package k8s
+
+import (
+	"context"
+	"fmt"
+
+	authv1 "k8s.io/api/authorization/v1"
+	v1 "k8s.io/client-go/kubernetes/typed/authorization/v1"
+)
+
+type selfSubjectRulesService struct {
+	client v1.AuthorizationV1Interface
+}
+
+func newSelfSubjectRulesService(client v1.AuthorizationV1Interface) *selfSubjectRulesService {
+	return &selfSubjectRulesService{
+		client: client,
+	}
+}
+
+func (svc *selfSubjectRulesService) Create(ctx context.Context, ssrr *authv1.SelfSubjectRulesReview) (*authv1.SelfSubjectRulesReview, error) {
+
+	token := ctx.Value("token").(string)
+	result := svc.client.RESTClient().Post().
+		AbsPath("/apis/authorization.k8s.io/v1").
+		Resource("selfsubjectrulesreviews").
+		Body(ssrr).
+		SetHeader("Authorization", token).
```

Comment:
@pkosiec whatever is passed as authorization header (even some invalid tokens) it doesnt change the fact that the request is executed in the context of service account.